app/src/main/java/org/oppia/app/topic/conceptcard/ConceptCardViewModel.kt

```diff
@@ -24,7 +24,7 @@ class ConceptCardViewModel @Inject constructor(
   @DefaultResourceBucketName private val resourceBucketName: String
 ) : ViewModel() {
   private lateinit var skillId: String
-  private lateinit var binding: ConceptCardFragmentBinding
+  private lateinit var view: TextView
```

Comment:
I actually think this will introduce a memory leak--view models shouldn't reference views directly. However, it's circumvented by ``@FragmentScope`` incorrectly recreating the view model. Suggest adding a TODO to update this view model to not depend on a view directly.

Reply:
Okay - Added a TODO.